commands/scp.go

```diff
@@ -148,6 +150,11 @@ func getInfoForScpArg(hostAndPath string, hostInfoLoader HostInfoLoader) (HostIn
 	}
 
 	args := []string{}
+	port, err := hostInfo.GetSSHPort()
+	if err == nil && port > 0 {
+		args = append(args, "-P", fmt.Sprintf("%v", port))
+	}
```

Comment:
I'm not sure, I had just google for it and found [that page](https://linuxacademy.com/blog/linux/ssh-and-scp-howto-tips-tricks/#specifying-a-port-with-scp) : 
> Also note that the -P option must be in front of the (remote) server. Why? Because scp also supports copying between two servers and therefore needs to know which server the -P option applies to.

But this is just a blog and by no mean official documentation for scp, so it might be wrong.

I don't have such a setup with custom ports to be able to test, looking at your output it seems like scp would try different ports in a row, could you try with this command : `scp -P 231 -P 232 root@138.68.61.69:./README.md root@138.68.50.80:./`.

If that works then my bad and the current PR is fine :)